doc/classes/RayCast2D.xml

```diff
@@ -69,13 +69,14 @@
 			<return type="Vector2" />
 			<description>
 				Returns the normal of the intersecting object's shape at the collision point, or [code]Vector2(0, 0)[/code] if the ray starts inside the shape and [member hit_from_inside] is [code]true[/code].
+				[b]Warning:[/b] Check that [method is_colliding] returns [code]true[/code] before calling this method to ensure the returned normal is meaningful and up-to-date.
```

Comment:
Warnings are reserved for more "dangerous" operations. Saying both "_meaningful_" and "_up-to-date_" is a bit redundant, and the former is too up to interpretation, so it could be changed.
```suggestion
				[b]Note:[/b] Check that [method is_colliding] returns [code]true[/code] before calling this method to ensure the returned normal is valid and up-to-date.
```